[OSDOCS#16961]: AUTH-3: Roles, Security Contexts, and Access Policies-CQA-1 (#115746)
openshift/openshift-docs: modules/security-context-constraints-psa-sync-exclusions.adoc

[id="security-context-constraints-psa-sync-exclusions_{context}"]
= Pod security admission synchronization namespace exclusions

[role="_abstract"]

> If you use pod security admission synchronization, be aware that system-created namespaces are permanently excluded from synchronization.

Reviewer: ```suggestion
If you use pod security admission synchronization, the system-created namespaces are permanently disabled from synchronization.
```

[id="security-context-constraints-psa-sync-exclusions_{context}"]
= Pod security admission synchronization namespace exclusions

[role="_abstract"]

> If you use pod security admission synchronization, be aware that system-created namespaces are permanently excluded and user-created `openshift-*` namespaces are excluded by default, though you can re-enable synchronization on them.

Reviewer: This should replace what is in ifndef::openshift-dedicated,openshift-rosa[], otherwise the ROSA and Dedicated docs will show information that it should not.
